src/html5-qrcode-scanner.ts

```diff
@@ -199,15 +206,39 @@ export class Html5QrcodeScanner {
     public constructor(
         elementId: string,
         config: Html5QrcodeScannerConfig | undefined,
+        verbose: boolean | undefined);
+
+    /**
+     * Creates instance of this class.
+     *
+     * @param element The HTML DOM element.
+     * @param config Extra configurations to tune the code scanner.
+     * @param verbose - If true, all logs would be printed to console. 
+     */
+    public constructor(
```

Comment:
For my information, what does this constructor overload with no implementation do in typescript?

Reply:
The idea was to have multiple constructors with a different signature each.
So in order to implement this constructor overloads, we couldn't have two constructors with bodies. You have to create the constructors signatures and then a global constructor that will handle the whole logic.
An example can be found here: https://timmousk.com/blog/typescript-multiple-constructors/

The purpose of adding multiple constructor is having the following IntelliSense suggestions:
![image](https://user-images.githubusercontent.com/27914994/222096823-0e4666f8-d0aa-40f3-9b72-ea86f3521e7a.png)
![image](https://user-images.githubusercontent.com/27914994/222097106-4d19b704-fc2f-4067-90fa-351528a58b88.png)

Both solutions will work and I can remove this overloading if you prefer having one constructor with the suggested signature:
`constructor(elementOrElementId: string | HTMLElement, config: Html5QrcodeFullConfig | undefined);`

Waiting for your response to resolve the rest of the comments